src/fees/FeeGenerator.sol

```diff
@@ -122,13 +96,39 @@ abstract contract FeeGenerator is Ownable {
 
         // Copy tempFees to fees
         IParam.Fee[] memory fees = new IParam.Fee[](realFeeLength);
-        for (uint256 i = 0; i < realFeeLength; ++i) {
+        for (uint256 i; i < realFeeLength; ++i) {
             fees[i] = tempFees[i];
         }
 
         return fees;
     }
 
+    /// @notice Set fee calculator contracts
```

Comment:
Jeff's [PR](https://github.com/dinngo/composable-router-contract/pull/70/files#diff-4c5c1b304f84735526561571301bebb623d96d819fe3f19fd7b294cde2df7d79) has move the two functions upward.

Reply:
Sorry, it must be the wrong adjustment when rebasing PR

Reply:
Still noticing a difference in `FeeGenerator.sol` file changed, the two functions should be moved to line 37.

Reply:
Sorry, Fixed it.